Gordon360/Utilities/ImageUtils.cs

```diff
@@ -20,7 +21,11 @@ public static void DeleteImage(string imagePath)
         {
             try
             {
-                File.Delete(imagePath);
+                // remove server address from imagePath, otherwise Student News image not found
```

Comment:
This file is not specific to student news, so it shouldn't necessarily reference an issue with student news without more explanation.